<launch>
    
    <include file="$(find mrover)/launch/jetson.launch"/>
    <!--
      ============
      Localization
      ============
    -->
    <rosparam command="load" file="$(find mrover)/config/localization.yaml"/>
    <rosparam command="load" file="$(find mrover)/config/ekf.yaml"/>

    <!-- passthrough filter to replace EKF in datapath -->
    <node pkg="mrover" type="passthrough_filter.py" name="passthrough_filter"/>

    <!-- node to linearize GPS from geodetic to ENU cartesian coords -->
    <node name="gps_linearization" pkg="mrover" type="gps_linearization.py" output="screen"/>

    <param name="map_frame" value="none"/>
    <param name="odom_frame" value="map"/>
    <param name="base_link_frame" value="base_link"/>
    <param name="world_frame" value="map"/>
    <param name="use_odom_frame" value="false"/>
</launch>
